library/cwm/src/lib/cwm/widget.rb

```diff
@@ -602,13 +602,37 @@ class RadioButtons < AbstractWidget
     include ItemsSelection
     abstract_method :label
 
+    # @!method vspacing
+    #   @return [Fixnum] space between the options
+
```

Comment:
please mention here also method hspacing, as there is usually no other way to know which methods are useful.